frontend/homework/14/src/homepage/Item/ItemPage.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useContext } from 'react';
+import { useParams,useNavigate } from 'react-router-dom';
+import { ItemsContext } from '../../App';
+import searchIcon from '../../assets/search-icon.svg';
+import {ItemParams} from '../../interfaces/Item'
+
+export function ItemPage() {
+    const { id } = useParams<ItemParams>();
+    const { items } = useContext(ItemsContext);
+    const item = items.find(item => item.id === parseInt(id!, 10));
+
+    if (!item) {
+        return <div>Item not found</div>;
+    }
+
+    let navigate = useNavigate();
+    const handleBack = () => {
+        navigate(-1);
+    };
+
+    const itemPageStyles: { [key: string]: React.CSSProperties } = {
+        body: {
+            backgroundColor: 'rgb(236, 236, 236)',
+            margin: 0,
+            padding: 0,
+            fontFamily: "'Segoe UI', Tahoma, Geneva, Verdana, sans-serif",
+        },
+        itemTitle: {
+            fontSize: '28px',
+            fontWeight: 'bold',
+            marginBottom: '10px',
+        },
+        productDescriptionTitle: {
+            fontSize: '24px',
+            fontWeight: 'bold',
+            marginBottom: '10px',
+        },
+        itemRightContainer: {
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'space-evenly',
+            fontSize: '20px',
+            width: '40vw',
+            height: '40vh',
+        },
+        itemPageContainer: {
+            height: '80vh',
+            backgroundColor: 'rgb(236, 236, 236)',
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            padding: '20px',
+            justifyContent: 'space-evenly',
+        },
+        itemImage: {
+            height: '40vh',
+            width: '40vw',
+            objectFit: 'contain',
+        },
+        itemDetailsItemPage: {
+            display: 'flex',
+            justifyContent: 'space-evenly',
+            width: '80vw',
+            maxHeight: '90vh',
+            alignItems: 'center',
+        },
+        bold: {
+            fontWeight: 'bold',
+        },
+        backToHomeBtn: {
+            backgroundColor: '#007bff',
+            color: 'white',
+            border: 'none',
+            borderRadius: '10px',
+            padding: '10px',
+            fontSize: '16px',
+            cursor: 'pointer',
+        },
+    };
+     const headerStyles: { [key: string]: React.CSSProperties } = {
```

Comment:
Since this is too granular, lets not put this inside a component. Using a global theme is better.